.github/workflows/run_cell-type-ewings.yaml

```diff
@@ -0,0 +1,68 @@
+# This is a workflow to run the cell-type-ewings module
+# Analysis modules are run based on three triggers:
+# - Manual trigger
+# - On pull requests where code in the module has changed
+# - As a reusable workflow called from a separate workflow which periodically runs all modules
+
+name: Run cell-type-ewings analysis module
+env:
+  MODULE_PATH: analyses/cell-type-ewings
+  AWS_DEFAULT_REGION: us-east-1
+
+concurrency:
+  # only one run per branch at a time
+  group: cell-type-ewings-${{ github.ref }}
+  cancel-in-progress: true
+
+on:
+  workflow_dispatch:
+  workflow_call:
+  pull_request:
+    branches:
+      - main
+    paths:
+      - analyses/cell-type-ewings/**
+jobs:
+  run-module:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: bash -el {0}
+
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v4
+
+      - name: Download test data
+        run: ./download-data.py --test-data --format "sce,anndata" --samples SCPCS000490
+
+      - name: Set up R
+        uses: r-lib/actions/setup-r@v2
+        with:
+          r-version: 4.4.0
+          use-public-rspm: true
+
+      - name: Set up pandoc
+        uses: r-lib/actions/setup-pandoc@v2
+
+      - name: Install additional system dependencies
+        run: |
+          sudo apt-get install -y libcurl4-openssl-dev libglpk40 jags
+
+      - name: Set up renv
+        uses: r-lib/actions/setup-renv@v2
+        with:
+          working-directory: ${{ env.MODULE_PATH }}
+
+      - name: Set up conda
+        uses: conda-incubator/setup-miniconda@v3
+
+      - name: Install and activate locked conda environment
+        run: |
+          conda install conda-lock
+          conda-lock install --name openscpca-cell-type-ewings ${MODULE_PATH}/conda-lock.yml
```

Comment:
I see that in your script you are using `conda run`
https://github.com/AlexsLemonade/OpenScPCA-analysis/blob/8937a0aed3e36837b2f785e85dbf38a9dbe705a2/analyses/cell-type-ewings/annotate-tumor-cells.sh#L136

I think this is probably a bit of overhead and specificity that should not be needed. If you activate the conda environment before running the script, would this work the same way? The only way I would potentially see a problem would be if your conda environment also included an R installation. Is that the case? 

If not, I would probably leave the `name` here as `test` and modify the script to call the component scripts directly, without `conda run`.

Reply:
> I think this is probably a bit of overhead and specificity that should not be needed. If you activate the conda environment before running the script, would this work the same way? The only way I would potentially see a problem would be if your conda environment also included an R installation. Is that the case?

No I don't have R in the conda environment right now, which is why I couldn't just activate the environment and then run the script. I can add R into the environment and try again?

Reply:
It should use the system R version if there is no R in the conda environment, which is what you want! There should not be any other changes needed.

Reply:
But to run it locally you would need to activate the environment and restore renv before being able to run the whole script. I just thought it was simpler to keep the R and conda things separate because I was getting issues just trying to run it locally, regardless of GHA.

Reply:
I would expect the following commands run in the module directory to set everything up as needed:

```
Rscript -e "renv::restore()"
conda activate openscpca-cell-type-ewings

./annotate-tumor-cells.sh
```

When I do that, it is still using the system R and the correct conda environment. The renv restore is needed either way, and the conda environment should not interfere. What errors were you running into?

Reply:
```
(base) allyhawkins@allys-work-mac cell-type-ewings % Rscript -e "renv::restore()"
- The library is already synchronized with the lockfile.
(base) allyhawkins@allys-work-mac cell-type-ewings % conda activate openscpca-cell-type-ewings
(openscpca-cell-type-ewings) allyhawkins@allys-work-mac cell-type-ewings % ./annotate-tumor-cells.sh 
/Users/allyhawkins/Documents/ALSF/forked_repos/OpenScPCA-analysis/analyses/cell-type-ewings/results/annotate_tumor_cells_output
Starting workflow for SCPCS000490, SCPCL000822
Saving cell references...
- One or more packages recorded in the lockfile are not installed.
- Use `renv::status()` for more details.
- One or more packages recorded in the lockfile are not installed.
- Use `renv::status()` for more details.
Error in library(SingleCellExperiment) : 
  there is no package called ‘SingleCellExperiment’
Calls: suppressPackageStartupMessages -> withCallingHandlers -> library
Execution halted
```